packages/swapper/src/swappers/ArbitrumBridgeSwapper/endpoints.ts

```diff
@@ -112,6 +112,9 @@ export const arbitrumBridgeApi: SwapperApi = {
     if (!step) throw new Error(`No hop found for stepIndex ${stepIndex}`)
 
     const { buyAsset, sellAsset, sellAmountIncludingProtocolFeesCryptoBaseUnit } = step
+
+    if (!isExecutableTradeQuote(tradeQuote)) throw new Error('Cannot execute a trade rate')
```

Comment:
Maybe something more generic, could even mention no receive address or no wallet verbiage for more context. Reason I comment is no being able to execute a trade rate wouldn't really be accurate once rate and quote are split. We will always be sending a trade rate "quote" at that point and it will be able to be executed given we have a receive address or more accurately a wallet is connected.
```suggestion
    if (!isExecutableTradeQuote(tradeQuote)) throw new Error('Unable to execute trade')
```

Reply:
As discussed, not 100% set on the split yet, but regardless, a more generic error does make sense! 5cb17e941a